### PR TITLE
Add GraphQLTag model

### LINKED ISSUE #70: Unable to map ClusterTaggableManager
Hi,

I'm using the tag manager for wagtail to tag some of my posts however, I've noticed that grapple doesn't know how to interpret ClusterTaggableManager.

This is how my class looks
``` python
class FactSheetTag(TaggedItemBase):
    content_object = ParentalKey(
        'FactSheet',
        related_name='tagged_items',
        on_delete=models.CASCADE
    )
```

This is how my field looks
``` python
keywords = ClusterTaggableManager(through=FactSheetTag, blank=True)
```

This is the output from graphene's converter.py file:
```
Exception: Don't know how to convert the Django field fact_sheet.FactSheet.keywords (<class 'modelcluster.contrib.taggit.ClusterTaggableManager'>)
```

When previously trying to implement my own Graphene coverter for Wagtail, I used the following code.

``` python
class FlatTags(graphene.String):
    @classmethod
    def serialize(cls, value):
        tagsList = []
        for tag in value.all():
            tagsList.append(tag.name)
        return tagsList

@convert_django_field.register(ClusterTaggableManager)
def convert_tag_field_to_string(field, registry=None):
    return graphene.Field(FlatTags,
        description=field.help_text,
        required=not field.null)
```

Does a similar implementation already exist in wagtail-grapple that I have missed or should this be a feature request?

Also, how might I implement this feature in the case that ClusterTaggableManager is not supported at all?

### PR DESCRIPTION
**Pull request by ruisaraiva19**
Resolves #70 

This adds single and plural tags queries.
```graphql
{
  tag(id: 1) {
    id
    name
  }
}
```

```graphql
{
  tags {
    id
    name
  }
}
```

Using it on a model:
```python
from django.db import models
from modelcluster.fields import ParentalKey
from modelcluster.contrib.taggit import ClusterTaggableManager
from taggit.models import TaggedItemBase
from wagtail.admin.edit_handlers import FieldPanel
from wagtail.core.models import Page
from grapple.models import GraphQLTag


class BlogPageTag(TaggedItemBase):
    content_object = ParentalKey(
        "BlogPage", related_name="tagged_items", on_delete=models.CASCADE
    )


class BlogPage(Page):
    # other fields
    tags = ClusterTaggableManager(through=BlogPageTag, blank=True)

    content_panels = Page.content_panels + [
        # other panels
        FieldPanel("tags"),
    ]

    graphql_fields = [
        # other graphql fields
        GraphQLTag("tags"),
    ]
```

### CODE REVIEW

File: docs/general-usage/model-types.rst [hunk 1]
```diff
@@ -464,3 +464,21 @@ GraphQLPage
...
             layers deep and Grapple will handle the querying for you through multiple models.
+
+
+GraphQLTags
+-----------
+
+.. module:: grapple.models
+.. class:: GraphQLTags(field_name: str, **kwargs)
+
+    Use this field type to serialize a ClusterTaggableManager.
```
Comment (zerolab) [inline docs/general-usage/model-types.rst:475]:
```suggestion
    Use this field type to serialize a ``ClusterTaggableManager`` field.
```

File: docs/general-usage/model-types.rst [hunk 2]
```diff
@@ -464,3 +464,21 @@ GraphQLPage
         * ``source`` (string)
             You can pass a source string that is an attribute or method on the model itself. It can also be several
             layers deep and Grapple will handle the querying for you through multiple models.
+
+
+GraphQLTags
```
Comment (zerolab) [inline docs/general-usage/model-types.rst:469]: Should this be `GraphQLTag`?

**Review by zerolab — COMMENTED** [review]
A couple of minor things, but excellent work!

File: docs/general-usage/model-types.rst (hunk 2 above)
Comment (ruisaraiva19) [reply docs/general-usage/model-types.rst:469]: Yes, good catch.

**Comment by ruisaraiva19** [comment]
> A couple of minor things, but excellent work!

@zerolab All updated!

**Review by zerolab — APPROVED** [review]

**Pull request merged**